precompiles/randomness/Randomness.sol

```diff
@@ -74,6 +74,11 @@ interface Randomness {
     /// @custom:selector 81797566
     function relayEpochIndex() external view returns (uint64);
 
+    /// Return the deposit required to perform a request
```

Comment:
It doesn't return the deposit. I think it should be rephrase as return the amount of deposit required...